### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Monty7**
Merging to master

### CODE REVIEW

**Comment by Paden** [comment]
You didn't add comments to the top of your methods

**Comment by Monty7** [comment]
@Paden added comment

File: app.js [hunk 1]
```diff
@@ -1,14 +1,121 @@
-var Questions = function() {
-    this.question = ["What drink would you like?"];
+//Montensia Banks
+//This app displays drink ingredients from users input of drink types.
 
+/**
+ * Bartender responsbile for asking drink types
```
Comment (navkelley) [inline app.js]: typo in comment

File: app.js [hunk 2]
```diff
@@ -1,14 +1,121 @@
...
+ * @param  {number} min lowest number
+ * @param  {number} max hightest number
+ * @return {number} A number between min and max     
+ */
+Pantry.prototype.randomized = function(min, max) {
+        min = Math.ceil(min);
+        max = Math.floor(max);
+        return Math.floor(Math.random() * (max - min + 1)) + min;
+    }
+    /**
```
Comment (navkelley) [inline app.js]: Spacing should be consistent even in commenting.

**Comment by Monty7** [comment]
@navkelley Thank You for pointing it out. I was able to fix the styling and misspelling.

**Comment by Paden** [comment]
Good work @navkelley  and @Monty7

**Pull request merged**